src/huggingface_hub/inference/_providers/hf_inference/text_to_image.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, Optional, Union
```

Comment:
I feel that `hf-inference` modules will likely to be duplicate of each other all the time. Not an issue for this PR, we can refactor that later.